src/Value/Message.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace KafkaRest\Value;
+
+use JMS\Serializer\Annotation as Serializer;
+
+final class Message
+{
+    /**
+     * @Serializer\Type("string")
+     * @Serializer\SkipWhenEmpty()
+     * @var string
```

Comment:
it's on many places but when I create new instance, it has null value

```suggestion
     * @var string|null
```

bug?